app/main.py

```diff
@@ -2,0 +1,32 @@
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> None:
+        def __repr__(self) -> str:
+            return f"{{Name: {self.name}, Health: {self.health}, Hidden: {self.hidden}}}"
```

Comment:
As per the checklist, you should use double quotes instead of single quotes for string literals.